mods/d2k/sequences/structures.yaml

```diff
@@ -22,6 +22,8 @@ wall:
 		Offset: -30,-24
 
 medium_gun_turret:
+	null: DATA.R8
```

Comment:
If we can't avoid this, please rename it to `empty` or `invisible`. `null` feels very out of place in our yaml.

Reply:
Sure.